internal/cache/file/cache_handler_test.go

```diff
@@ -166,71 +169,70 @@ func doesFileExist(filePath string) bool {
 		return false
 	}
 
-	AssertEq(nil, err)
+	require.NoError(chrT.T(), err)
 	return false
 }
 
 func (chrT *cacheHandlerTest) Test_createLocalFileReadHandle_OnlyForRead() {
 	readFileHandle, err := chrT.cacheHandler.createLocalFileReadHandle(chrT.object.Name, chrT.bucket.Name())
 
-	ExpectEq(nil, err)
+	assert.NoError(chrT.T(), err)
 	_, err = readFileHandle.Write([]byte("test"))
-	ExpectNe(nil, err)
-	ExpectTrue(strings.Contains(err.Error(), "bad file descriptor"))
+	assert.ErrorContains(chrT.T(), err, "bad file descriptor")
 }
 
 func (chrT *cacheHandlerTest) Test_cleanUpEvictedFile() {
 	fileDownloadJob := chrT.getDownloadJobForTestObject()
 	fileInfo := chrT.cache.LookUp(chrT.fileInfoKeyName)
 	fileInfoData := fileInfo.(data.FileInfo)
 	jobStatusBefore := fileDownloadJob.GetStatus()
-	AssertEq(jobStatusBefore.Name, downloader.NotStarted)
+	require.Equal(chrT.T(), jobStatusBefore.Name, downloader.NotStarted)
 	jobStatusBefore, err := fileDownloadJob.Download(context.Background(), int64(util.MiB), false)
-	AssertEq(nil, err)
-	AssertEq(jobStatusBefore.Name, downloader.Downloading)
+	require.NoError(chrT.T(), err)
+	require.Equal(chrT.T(), jobStatusBefore.Name, downloader.Downloading)
 
 	err = chrT.cacheHandler.cleanUpEvictedFile(&fileInfoData)
 
-	ExpectEq(nil, err)
+	assert.NoError(chrT.T(), err)
 	jobStatusAfter := fileDownloadJob.GetStatus()
-	ExpectEq(jobStatusAfter.Name, downloader.Invalid)
-	ExpectFalse(doesFileExist(chrT.downloadPath))
+	assert.Equal(chrT.T(), jobStatusAfter.Name, downloader.Invalid)
```

Comment:
The second argument is what's expected and the third argument is what's actual. This helps understand the error messages better: "failed because expected $this but found $that". However, here and at a few other places the order seems to have been reversed.

Reply:
This is not reversed in this PR and was in reversed format before as well. I can correct this and maybe some more but I can't guarantee to correct all in scope of this PR.

Reply:
Thanks!